### DISCUSSION TITLE
Problem using with laravel-sluggable

### DISCUSSION THREAD

**Discussion by HungryBus** [spatie/laravel-translatable#423] [Q&A]
Probably a bug, since when using this package together with laravel-sluggable, and model has Slug field generated from the translatable field, slug is being translated to the last translated attribute:

```
class Field extends Model {
  use HasSlug, HasTranslations;

  public function getSlugOptions(): SlugOptions
  {
    return SlugOptions::create()
      ->generateSlugsFrom('field_name')
      ->usingLanguage('en')
      ->saveSlugsTo('field_slug');
  }
```

Let's say field has been created without translation with field_name = 'generation' and auto-generated slug of 'generation';

Then 
```
$field->setTranslation('field_name', 'lv', 'some translation');
$field->save()
```

will change entry's slug to 'some-translation';

I guess that the only solution is to prevent slugs from regenerating on update;

**Comment by automat64** [comment]
Are you by any chance using filament with spatie-laravel-translatable-plugin?  
I have the same problem but only when updating through filament panel. Both slugs are updated to the current editing language tab.
Using tinker for manipulating the model updates slugs correctly.

**Reply by mouuuton** [reply]
Hi, same issue here, did you find a fix?

**Reply by automat64** [reply]
I can't exactly remember, but I think i used this: https://larazeus.com/translatable
